mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx

```diff
@@ -241,6 +235,7 @@ const AITransformationTaskpane = (props: AITransformationTaskpaneProps): JSX.Ele
                     previousFailedCompletions.push([completionOrError.completion, possibleError])
                 } else {
                     setTaskpaneState({type: 'default'});
+                    setSuccessfulExecutionSinceOpen(true); // Mark a successful execution
```

Comment:
Since this gets set after the `previousParamsAndResults` is set and in a race with `taskpaneState`, the first render does not have the recon turned on. 

I fixed this by adding `successfulExecutionSinceOpen` to the useEffect triggers.